tests/additional_outputs_02/screen-output

```diff
@@ -8,24 +8,14 @@ Number of active cells: 4 (on 2 levels)
 Number of degrees of freedom: 84 (50+9+25)
 
 *** Timestep 0:  t=0 seconds
-* create_additional_material_model_outputs() called
```

Comment:
why is this no longer called?

Reply:
Because previously if an assembler was in the list of assembler_objects create_additional_material_model_outputs() would be called for all equations, while now it is only called for the equations in which this assembler is actually used (in this case the Stokes equations). Since the Assembler can only be used in the Stokes equation (to successfully cast the scratch object), I can not reproduce the old output, but the new one is more reasonable anyway.